src/helperFunctions/install.py

```diff
@@ -70,12 +69,12 @@ def log_current_packages(packages: Tuple[str], install: bool = True):
 
 
 def _run_shell_command_raise_on_return_code(command: str, error: str, add_output_on_error=False) -> str:  # pylint: disable=invalid-name
-    output, return_code = execute_shell_command_get_return_code(command)
-    if return_code != 0:
+    cmd_p = subprocess.run(command, shell=True, stdout=PIPE, stderr=STDOUT, text=True)
```

Comment:
I would prefer better readable variable names like simply "process" instead of "cmd_p" (I know what that stands for but someone else who looks at the file at some point in the future maybe doesn't)